tests/integration/profiling/rule_based_profilers/test_run_profiler_notebook.py

```diff
@@ -43,3 +44,6 @@ def test_run_rbp_notebook(tmp_path):
     finally:
         with open(output_notebook_path, mode="w", encoding="utf-8") as f:
             nbformat.write(nb, f)
+
+    # clean up Expectations directory after running tests
+    shutil.rmtree(os.path.join(base_dir, "great_expectations/expectations/"))
```

Comment:
This is the magical line @alexsherstinsky sorry for missing it the first time around.